src/app/work-item/work-item-list/work-item-list-entry/work-item-list-entry.component.ts

```diff
@@ -5,6 +5,7 @@ import { AuthenticationService } from '../../../auth/authentication.service';
 import { Broadcaster } from '../../../shared/broadcaster.service';
 import { Logger } from '../../../shared/logger.service';
 
+import { ModalModule }       from 'ng2-modal';
```

Comment:
Do we need to import it here in the component? Because it already imported in work item module.